### PR TITLE
Change discord invite to a banner

### PR DESCRIPTION
**Pull request by Gideon9212**
To help direct people to support with a big banner.

### CODE REVIEW

**Comment by Gideon9212** [comment]
What it will look like. Due to no discord server icon atm, it will be blank until you add one.
image

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Portable builds are provided for AppImage on Linux, Windows, and macOS.
 
 ## Help & Support
 
-- Join the [Discord Server](https://discord.gg/hX4g537UNE) for now. 
+[![Support server invite](https://discordapp.com/api/guilds/1031648380885147709/widget.png?style=banner3)](https://discord.gg/hX4g537UNE)
```
Comment (piprett) [inline README.md]: I think "Support server invite" is not the best. The previous "Join the Discord Server" sounds better in my opinion.

**Review by piprett — CHANGES_REQUESTED** [review]
Small nitpick

**Review by AshtakaOOf — APPROVED** [review]
Pretty cool

File: README.md (hunk 1 above)
Comment (Gideon9212) [reply README.md]: I can change that if needed. It is a simple change.

**Comment by Gideon9212** [comment]
I updated that nitpick.
image

**Review by piprett — APPROVED** [review]

**Review by flowln — APPROVED** [review]
thank you! just pleases resolve the conflict :p

**Comment by Gideon9212** [comment]
Conflict resolved.

**Review by AshtakaOOf — CHANGES_REQUESTED** [review]
Use the old invite in case the vanity dies because of boosters

**Comment by Gideon9212** [comment]
Swaped back to old invite.

**Review by timoreo22 — APPROVED** [review]

**Pull request merged**